docs/developer-guide/syntax-reference.md

```diff
@@ -99,6 +99,31 @@ You can create your own aggregation logic by implementing a User Defined
 Aggregation Function (UDAF). For more information, see
 [UDAFs](../concepts/functions.md#udafs).
 
+Aggregations return results per partition. To get results across all partitions,
+you can use PARTITION BY with a fixed key, for example:
+
+```sql
+CREATE STREAM allplaycounts AS
+  SELECT "fixed_key" VARCHAR KEY, COUNT(*) FROM
```

Comment:
I think `VARCHAR KEY` should be removed here. Type and key specification are only for CT/CS statements. Also, single quotes instead of double quotes around `'fixed_key'`